src/metalnx-web/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -54,9 +54,11 @@
 	<bean id="ep403"
 		class="com.emc.metalnx.handler.Http403ForbiddenEntryPoint" />
 
-	<sec:http auto-config="true" create-session="always"
+	<sec:http auto-config="true" create-session="always" use-expressions="false"
```

Comment:
Please add a comment explaining why the new property is needed.

Reply:
done